clouddriver-elasticsearch/src/test/groovy/com/netflix/spinnaker/clouddriver/elasticsearch/events/CreateServerGroupEventHandlerSpec.groovy

```diff
@@ -0,0 +1,37 @@
+package com.netflix.spinnaker.clouddriver.elasticsearch.events
+
+import com.netflix.spinnaker.clouddriver.elasticsearch.ElasticSearchServerGroupTagger
+import com.netflix.spinnaker.clouddriver.orchestration.events.CreateServerGroupEvent
+import com.netflix.spinnaker.clouddriver.orchestration.events.OperationEvent
+import spock.lang.Specification
+import spock.lang.Subject
+import spock.lang.Unroll
+
+class CreateServerGroupEventHandlerSpec extends Specification {
+  def serverGroupTagger = Mock(ElasticSearchServerGroupTagger)
+
+  @Subject
+  def eventHandler = new CreateServerGroupEventHandler(serverGroupTagger)
+
+  @Unroll
+  void "should only handle DeleteServerGroupEvent"() {
+    given:
+    def operationEvent = Mock(OperationEvent) {
+      getType() >> { return OperationEvent.Type.SERVER_GROUP }
+      getAction() >> { return OperationEvent.Action.CREATE }
```

Comment:
That this case (type is `SERVER_GROUP`, action is `CREATE`) is not handled is a little confusing - I would kind of expect the event to be handled based on those two fields. But I guess it's good that the test shows it really has to be an event of that particular class.

Maybe update the name of the test to add that clarification? Or not, it's probably fine.